### ISSUE TITLE
`\!` escape to execute shell command does not work

### ISSUE THREAD

**Issue by levinotik** [dbcli/pgcli#618]
When trying to use `\!` to execute a shell command, it results in an error. For example:

```
my_db> \! pwd
syntax error at or near "\"
LINE 1: \! pwd
```

**Comment by amjith** [comment]
Yes. This would definitely be nice to have. Thanks for the request.

**Comment by j-bennet** [comment]
Additionally, if the user executes a special command that we don't support, it would be good to handle that gracefully ("This special command is not supported in pgcli yet."). What do you think @amjith?

**Comment by dbaty** [comment]
It's been implemented almost 5 years ago in dbcli/pgspecial#74. Demo in the latest version of pgcli (4.0.1):

```
$ pgcli
> \! pwd
/home/damien/dev/perso/pgcli
```
